backend/src/main/java/de/bund/digitalservice/ris/caselaw/config/DocumentationOfficeUserGroupsConfig.java

```diff
@@ -0,0 +1,100 @@
+package de.bund.digitalservice.ris.caselaw.config;
+
+import de.bund.digitalservice.ris.caselaw.adapter.DocumentationOfficeConfigUserGroup;
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DocumentationOfficeUserGroupsConfig {
+
+  /**
+   * Caution: Changing values will delete and recreate the affected group in our database. This
+   * means all references to the affected group will be deleted, e.g., the links of the affected
+   * user group to procedures will be cleared.
+   */
+  @Bean
+  public List<DocumentationOfficeConfigUserGroup> DocumentationOfficeConfigUserGroupConfig() {
```

Comment:
Violates naming convention as it is a method and contains two times "config" (?):
```suggestion
  public List<DocumentationOfficeConfigUserGroup> getDocumentationOfficeConfigUserGroups() {
```